### PR TITLE
Cob4 both arms ik planning

### PR DESCRIPTION
**Pull request by juandhv**
@ChamzasKonstantinos This PR includes the test of the new planning group to solve IK and planning queries for both_arms with the Care-O-Bot4.

### CODE REVIEW

**Comment by zkingston** [comment]
Please rebase this branch against the master for robowflex so there aren't any weird merge issues.

**Comment by juandhv** [comment]
@zkingston there shouldn't be conflicts now.

File: robowflex_library/scripts/cob4_multi_target.cpp [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    MotionRequestBuilder request_both_arms(planner, BOTH_ARMS);
+    request_both_arms.setStartConfiguration(cob4->getScratchState());
+
+    // Create a motion planning request with a pose goal for both_arms
+    const auto &goal_pose_right = TF::createPoseQ(0.6, -0.26, 0.97, 1.0, 0.0, 0.0, 0.0);
+    const auto &goal_pose_left = TF::createPoseQ(0.38, 0.26, 0.78, 0.0, 0.707, 0.0, 0.707);
+
+    // Visualizing the target poses.
+    rviz.addTransformMarker("pose_right", "base_link", goal_pose_right);
+    rviz.addTransformMarker("pose_left", "base_link", goal_pose_left);
```
Comment (zkingston) [inline robowflex_library/scripts/cob4_multi_target.cpp:71]: Is it "base_link" in the Cob4?

File: robowflex_library/scripts/cob4_multi_target.cpp [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    // Create the default Care-O-Bot4 robot.
+    auto cob4 = std::make_shared<Cob4Robot>();
+    cob4->initialize();
+
+    // Load kinematics for the COB4 robot.
+    cob4->loadKinematics(BOTH_ARMS);
+
+    // Fold the arms.
+    cob4->setGroupState(RIGHT_ARM, {2.69, 1.70, -0.91, 1.50, -2.14, -2.35, 1.06});
+    cob4->setGroupState(LEFT_ARM, {-1.14, -1.50, 0.34, -1.50, 0.43, -1.56, -1.20});
```
Comment (zkingston) [inline robowflex_library/scripts/cob4_multi_target.cpp:44]: Can you just set them all at once with BOTH_ARMS?

File: robowflex_library/scripts/cob4_multi_target.cpp [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    planning_interface::MotionPlanResponse res = planner->plan(scene, request_both_arms.getRequest());
+    if (res.error_code_.val != moveit_msgs::MoveItErrorCodes::SUCCESS)
+        return 1;
+
+    // Publish the trajectory to a topic to display in RViz
+    rviz.updateTrajectory(res);
+
+    // Visualize resulting state.
+    auto left_arm_trajectory = std::make_shared<Trajectory>(res.trajectory_);
+    cob4->setGroupState(LEFT_ARM, left_arm_trajectory->vectorize().back());
```
Comment (zkingston) [inline robowflex_library/scripts/cob4_multi_target.cpp:100]: Whyu just the left arm? This should be for both.

**Review by zkingston — APPROVED** [review]
LGTM, just some small changes that once fixed we can merge in.

File: robowflex_library/scripts/cob4_multi_target.cpp (hunk 1 above)
Comment (juandhv) [reply robowflex_library/scripts/cob4_multi_target.cpp:71]: Yes, it is.

File: robowflex_library/scripts/cob4_multi_target.cpp (hunk 2 above)
Comment (juandhv) [reply robowflex_library/scripts/cob4_multi_target.cpp]: Done!

File: robowflex_library/scripts/cob4_multi_target.cpp (hunk 3 above)
Comment (juandhv) [reply robowflex_library/scripts/cob4_multi_target.cpp]: True, fixed!

**Comment by juandhv** [comment]
Thanks for the comments, @zkingston !

**Pull request merged**